alerting/build.gradle

```diff
@@ -98,6 +98,15 @@ integTest.dependsOn(bundle)
 integTest.getClusters().forEach{c -> c.plugin(project.getObjects().fileProperty().value(bundle.getArchiveFile()))}
 
 def _numNodes = findProperty('numNodes') as Integer ?: 1
+
+String notificationsVersion = "2.0.0"
+String notificationsQualifier = "rc1"
+String notificationsFilePath = "src/test/resources/notifications"
+String notificationsCoreFilePath = "src/test/resources/notifications-core"
+String notificationsOpenDistroPlugin = "opensearch-notifications-" + notificationsVersion + ".0-" + notificationsQualifier + ".zip"
+String notificationsCoreOpenDistroPlugin = "opensearch-notifications-core-" + notificationsVersion + ".0-" + notificationsQualifier + ".zip"
```

Comment:
NP: These can just be renamed to `notifications/notificationsCorePlugin` since they aren't OpenDistro plugins.